examples/resources/groundcover_dashboard/import.sh

```
@@ -0,0 +1 @@
+terraform import groundcover_dashboard.example <id>
```

Shellcheck (0.11.0)

[error] Couldn't parse this redirection. Fix to allow more checks.

(SC1073)